runtime/vm/jnimisc.cpp

```diff
@@ -792,11 +792,7 @@ getStringUTFChars(JNIEnv *env, jstring string, jboolean *isCopy)
 		gpCheckSetNativeOutOfMemoryError(currentThread, 0, 0);
 	} else {
 		JAVA_OFFLOAD_SWITCH_ON_WITH_REASON_IF_LIMIT_EXCEEDED(currentThread, J9_JNI_OFFLOAD_SWITCH_GET_STRING_UTF_CHARS, utfLength);
-		if (UDATA_MAX != copyStringToUTF8Helper(currentThread, stringObject, TRUE, J9_STR_NONE, utfChars, utfLength)) {
-			if (NULL != isCopy) {
-				*isCopy = JNI_TRUE;
-			}
-		}
+		copyStringToUTF8Helper(currentThread, stringObject, J9_STR_NONE, utfChars);
```

Comment:
Setting `isCopy` code is missing.

Reply:
Fixed in 1baa27d. I wonder why no tests failed due to this bug which was introduced by me. Thank you very much for catching this! I think we need more testing here.

Reply:
Opened https://github.com/eclipse/openj9/issues/1102 to track this.